__tests__/app.test.js

```diff
@@ -67,6 +67,41 @@ describe('GET /api/articles', () => {
 });
 describe('GET /api/articles/:article_id', () => {
     it('should respond with a status 200 and an author object ', () => {
-        
+        return request(app)
+            .get("/api/articles/3")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article).toStrictEqual(
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),

```

Comment:
This should be the actual article_id from the data instead of expect.any. This is the same for the other properties.